bpy-build.yaml

```diff
@@ -0,0 +1,13 @@
+addon_folder: MCprep_addon
+build_name: MCprep_addon
+
+install_versions:
+  - '3.5'
+  - '3.4'
+
+during_build:
+  default:
+    - [flake8 --extend-ignore W191 .]
```

Comment:
I see now this is included, which is good - but for some reason, I'm still getting W191 printouts (completely taking up my screen).